tests/integration/tls_tests/helpers.py

```diff
@@ -43,7 +54,7 @@ async def mongo_tls_command(ops_test: OpsTest) -> str:
     )
 
 
-async def check_tls(ops_test: OpsTest, unit: ops.model.Unit, enabled: bool) -> bool:
+async def check_tls(ops_test: OpsTest, unit: ops.model.Unit, enabled: bool, app_name=None) -> bool:
```

Comment:
Similar to the above, the app_name doesn't seem like it can be as an optional

Reply:
@Mehdi-Bendriss from what I can tell `app_name` can be None here, since in the line that it is used (`73`) we make the call `mongod_tls_check = await mongo_tls_command(ops_test, app_name=app_name)` and `mongo_tls_command` does not require the option `app_name` and will infer it in the case of `None`